mixer/cmd/build.go

```diff
@@ -163,6 +163,8 @@ var buildUpstreamFormatCmd = &cobra.Command{
 				fail(err)
 			}
 		}
+		fmt.Println(`NOTE: upstream bundle definitions may have changed,
```

Comment:
This prints every time, regardless of whether the bundle definitions have change?

Reply:
yes :O 

the only way to check I think would be to save a copy of the old ones and then diff the folders. Which I can do if that's a better time to print.